src/actions/background/BackgroundAction.ts

```diff
@@ -0,0 +1,20 @@
+import {IAction} from "../../interfaces/IAction";
+import Param from "../../parameters/Param";
+import ParamValue from "../../parameters/ParamValue";
+
+class BackgroundAction extends Param implements IAction {
+  constructor(backgroundType?: string) {
+    super('b', new ParamValue(backgroundType).setDelimiter('_'));
+  }
+
+  /**
+   * @memberOf Actions.Background
```

Comment:
This method is not actually a member of Actions.Background right?

A user cannot import {contract} from actions/background.

Reply:
But it is public. How should we add it to the docs?